config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  resources :users do
```

Comment:
- [ ] It will be great to ensure that you only generate the routes that are needed in the application. For the users, you should only generate `index` and `show` routes

Reply:
okay sir, ill work on it right away